docs/source/getting_started/teams.rst

```diff
@@ -0,0 +1,132 @@
+========
+Teams
+========
+
+Teams overview
+
+.. warning::
+    Teams are not available on all Citrine Platform deployments.
+
+    Please contact support for more information.
+
+Basic Team Use
+-----------------
+
+Most commonly, the first thing you will want to do after connecting to the Citrine Platform
+is to find a certain Team. Assume that you have created a :class:`~citrine.citrine.Citrine`
+client object named ``citrine``. To list all of the Teams that you are a member of, use the
+``list`` command.
+
+.. code-block:: python
+
+    citrine.teams.list()
+
+To retrieve a Team that you are a member of, either find the Team in the list:
+
+.. code-block:: python
+
+    team_name = "Team A"
+    all_teamss = citrine.teams.list()
+    team_a = next((team for team in all_teams if team.name == team_name), None)
+
+or get the Team by name using a helper method:
+
+.. code-block:: python
+
+    from citrine.seeding.find_or_create import find_or_create_team
+    team_name = "Team A"
+    team_a = find_or_create_team(team_collection=citrine.teams, team_name=team_name)
+
+or get it by unique identifier:
+
+.. code-block:: python
+
+    team = citrine.teams.get("baaa467e-1758-43a8-97c7-76e569d0dcab")
+
+Managing Users
+--------------
+
+Admin users can manage permissions of Users in Teams.
+
+There are three types of access permissions a user can have on a team: READ, SHARE and WRITE.
+
+- READ allows a user to view resources in a Team.
+- WRITE allows them to modify those resources.
+- SHARE allows them to publish those resources to other Teams.
+
+There are several methods for managing teams, Users, and user membership in teams
+
+
+Listing Users in a Team
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users in a Team can be listed using the :func:`~citrine.resources.team.Team.list_members` method.
+The ``TeamMember`` array returned from this method has the user's access permissions in the Team
+as well as a copy of the User and Team objects.
+
+.. code-block:: python
+
+     team = citrine.teams.register(name="Your Team")
+
+     # List Members of a team
+     team_members = team.list_members()
+
+     # See their roles
```

Comment:
The comments refers to it as a role, but the field name is `action`, and it seems 1-to-1 with what project actions were.

Reply:
typo, this should be actions